internal/standalone/standalone.go

```diff
@@ -38,8 +38,12 @@ func GetInfo(id string) (Args, error) {
 		return info, err
 	}
 
-	// When debugging in vscode, write the file in `dist`
-	if standalone && strings.HasPrefix(filepath.Base(ex), "__debug_bin") {
+	// When debugging, write the file in `dist`
+	// VsCode names the file "__debug_bin"
+	vsCodeDebug := strings.HasPrefix(filepath.Base(ex), "__debug_bin")
+	// GoLand places it in: /tmp/GoLand/___XXgo_build_github_com_PACKAGENAME_pkg
```

Comment:
Do we know if this is consistent across platforms?

Reply:
On Mac OS I got:
```
/usr/local/go/bin/go build -o /private/var/folders/lx/k8zcwy851kzdrxrzjwsv34gr0000gn/T/GoLand/___go_build_PACKAGENAME
```

So it should match anyways. Not sure about Windows, I will test shortly.